mewarpx/mewarpx/diags_store/checkpoint_diagnostic.py

```diff
@@ -0,0 +1,43 @@
+"""Class for installing a checkpoint diagnostic"""
+
+from mewarpx.mwxrun import mwxrun
+from mewarpx.diags_store.diag_base import WarpXDiagnostic
+
+from pywarpx import picmi
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CheckPointDiagnostic(WarpXDiagnostic):
+    def __init__(self, diag_steps, name="checkpoint", write_dir=".", **kwargs):
```

Comment:
Rather than have the default value string literal `"checkpoint"` in multiple places, please name a variable somewhere and use that in both sites, writing and reading checkpoints, so it's clear they're a matched pair.